### ISSUE TITLE
utf8bom error when create and open files?

### ISSUE THREAD

**Issue by Oooocean** [theia-ide/theia-apps#352]
1. centos host: docker run  -u $UID -i -p  3000:3000 -v "$(pwd):/home/project" theiaide/theia
2. mac client: open browser and create file named "xxx.txt"

Here are ouputs:
```
root INFO [nsfw-watcher: 19] Started watching: /home/project/xxx.txt
root ERROR Request resolveContent failed with error: Encoding not recognized: 'utf8bom' (searched as: 'utf8bom') Error: Encoding not recognized: 'utf8bom' (searched as: 'utf8bom')
    at Object.getCodec (/home/theia/node_modules/@theia/filesystem/node_modules/iconv-lite/lib/index.js:106:23)
    at Object.getDecoder (/home/theia/node_modules/@theia/filesystem/node_modules/iconv-lite/lib/index.js:127:23)
    at Object.decode (/home/theia/node_modules/@theia/filesystem/node_modules/iconv-lite/lib/index.js:40:25)
    at FileSystemNode.<anonymous> (/home/theia/node_modules/@theia/filesystem/lib/node/node-filesystem.js:174:41)
    at step (/home/theia/node_modules/@theia/filesystem/lib/node/node-filesystem.js:59:23)
    at Object.next (/home/theia/node_modules/@theia/filesystem/lib/node/node-filesystem.js:40:53)
    at fulfilled (/home/theia/node_modules/@theia/filesystem/lib/node/node-filesystem.js:31:58)
```

**Comment by vince-fugnitto** [comment]
@Oooocean thank you for the issue, when the error is reported is there any loss of functionality?
Are you still able to create/open/save files?

**Comment by vince-fugnitto** [comment]
The issue comes from [iconv-lite](https://github.com/ashtuchkin/iconv-lite), and they have a section regarding [bom-handling](https://github.com/ashtuchkin/iconv-lite#bom-handling). Maybe there is something missing from the frameworks end.

**Comment by akosyakov** [comment]
It will be fixed in https://github.com/eclipse-theia/theia/pull/7908 See https://github.com/eclipse-theia/theia/blob/33499f860be69f4c44c85fc2378a145ead839fea/packages/filesystem/src/browser/encoding-service.ts#L163-L168

**Comment by vince-fugnitto** [comment]
Fixed thanks to eclipse-theia/theia#7908 👍
